### ISSUE TITLE
EditText inside DialogSheet uneditable

### ISSUE THREAD

**Issue by RasyiidWho** [marcoscgdev/DialogSheet#25]
Hi marcoscgdev,

I have recently update DialogSheet from 2.0.9 to 2.1.0 it was a great inprovement (good work dev :D). But, I found a problem in this version (2.1.0) that EditText (inside dialog, when using setView) are now uneditable, I can paste some text from clipboard, and I got warning from logcat:

`W/TextView: TextView does not support text selection. Selection cancelled.`

I also try set false or true in useNewStyle but that's gives me nothing. I can give my logcat if it's needed.

Thankyou :)

**Comment by marcoscgdev** [comment]
Apparently the issue has been resolved. I will publish a new release with this and other changes in a few days.

While I prepare the new release, you can use this build, which has this problem fixed:
```groovy
implementation 'com.github.marcoscgdev:DialogSheet:5a4dc5d656'
```

Thank you for reporting this issue 😄

**Comment by marcoscgdev** [comment]
Fixed in [version 2.1.1](https://github.com/marcoscgdev/DialogSheet/releases/tag/2.1.1).
